presto-native-execution/presto_cpp/main/SessionProperties.cpp

```diff
@@ -443,6 +443,23 @@ SessionProperties::SessionProperties() {
       false,
       QueryConfig::kScaleWriterMinProcessedBytesRebalanceThreshold,
       std::to_string(c.scaleWriterMinProcessedBytesRebalanceThreshold()));
+
+  addSessionProperty(
+      kTableScanScaledProcessingEnabled,
+      "If set to true, enables scaled processing for table scans.",
+      BOOLEAN(),
+      false,
+      QueryConfig::kTableScanScaledProcessingEnabled,
+      std::to_string(c.tableScanScaledProcessingEnabled()));
+
+  addSessionProperty(
+      kTableScanScaleUpMemoryUsageRatio,
+      "Controls the ratio of available memory that can be used for scaling up table scans. "
+      "The value is in the range of (0, 1].",
```

Comment:
I assume, it should be a bracket, and that 0 is included in the range, or if not, the other description in nativeSessionPropertyProvider should match this one.
```suggestion
      "The value is in the range of [0, 1].",
```

Reply:
Yes 0 is included, updated this